infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -587,7 +596,7 @@ private void runFailureDetectorTask() {
      */
     private boolean isCurrentLayoutSlotUnFilled(PollReport pollReport) {
         boolean result = pollReport.getOutOfPhaseEpochNodes().keySet()
-                .containsAll(serverContext.getManagementLayout().getLayoutServers());
+                .containsAll(getManagementLayout().getLayoutServers());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "getManagementLayout()" can return null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)